editor/editor_help_search.cpp

```diff
@@ -161,7 +161,9 @@ void EditorHelpSearch::popup_dialog() {
 void EditorHelpSearch::popup_dialog(const String &p_term) {
 	// Restore valid window bounds or pop up at default size.
 	Rect2 saved_size = EditorSettings::get_singleton()->get_project_metadata("dialog_bounds", "search_help", Rect2());
-	if (saved_size != Rect2()) {
+	Rect2 screen_size = DisplayServer::get_singleton()->screen_get_usable_rect(get_window()->get_current_screen());
```

Comment:
```suggestion
	Rect2 screen_rect = DisplayServer::get_singleton()->screen_get_usable_rect(get_window()->get_current_screen());
```
To clarify it's size *and* position

Reply:
Should I change the line above, too? From `saved_size` to `saved_rect`.

Reply:
Makes sense, could use "bounds" as that's what the meta is called